src/utils/builder.ts

```diff
@@ -389,13 +389,15 @@ export function setTransitionCondition(
       (transition: Transition) => transition.id === transitionId
     );
 
-    if (foundTransition)
+    if (foundTransition) {
+      const cql = criteria.statement;
       foundTransition.condition = {
         description: description,
-        cql: criteria.statement,
+        cql: cql,
```

Comment:
I think you can just revert this to how it was before, `cql: criteria.statement` so you don't need to set the unnecessary cql const on line 393.